Modules/xSharePoint/DSCResources/MSFT_xSPFarmAdministrators/MSFT_xSPFarmAdministrators.psm1

```diff
@@ -0,0 +1,224 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [parameter(Mandatory = $true)] [System.String] $Name,
+        [parameter(Mandatory = $false)] [System.String[]] $Members,
+        [parameter(Mandatory = $false)] [System.String[]] $MembersToInclude,
```

Comment:
@kilasuit - I see what you mean and I agree. But what about a situation where you want to add an account to the Farm Administrators group, but do not "care" about any other users in the group? Or if you want to prevent a specific user to be added to the Farm Administrator group, but the rest of the members are flexible. With these two properties you have the flexibility to do that, that is why I added them. You can use DSC to "Make it so" :-)

The code contains a check to prevent you from using Members and any of the other two parameters at the same time.